pkgs/tools/system/efivar/default.nix

```diff
@@ -12,6 +12,13 @@ stdenv.mkDerivation rec {
     rev = version;
     sha256 = "1z2dw5x74wgvqgd8jvibfff0qhwkc53kxg54v12pzymyibagwf09";
   };
+  patches = [
+    (fetchurl {
+      name = "r13y.patch";
+      url = "https://patch-diff.githubusercontent.com/raw/rhboot/efivar/pull/133.patch";
```

Comment:
Great work! Nice to get r13y another step forward :D

Two note (for both @delroth and @mmahut):

1) generally we should use fetchpatch over fetchurl for patches, as it makes the patches a bit more stable
2. these pull request .patch URLs are quite unstable. As the PR branch changes, or the branch it is merged in to changes, this patch will change contents and the hash will become invalid. In this case fetching the commit's patch directly (https://github.com/delroth/efivar/commit/b81e2f7d72d79b941f71962819606b802e6e835c.patch) would be better.